src/sbt-test/docker/test-layer-groups/changes/nolayers.sbt

```diff
@@ -1 +1 @@
-dockerLayerGrouping in Docker  := (_ => None)
+dockerGroupLayers in Docker := PartialFunction.empty
```

Comment:
It would be nice to reflect this in the docs as well:

https://github.com/sbt/sbt-native-packager/blame/master/src/sphinx/formats/docker.rst#L147-L154